### DISCUSSION TITLE
No Golems (spawning)

### DISCUSSION THREAD

**Discussion by Primitive-Human** [ACGaming/UniversalTweaks#51] [Tweak Suggestions]
Basically the [No Golems](https://www.curseforge.com/minecraft/mc-mods/no-golems) mod, which disables the spawning of iron golems, snow golems and withers by **building** them. I believe these mobs deserve a better way to create them, hence why this vanilla method should be disabled. 

Otherwise Mojang should just implement the [Accurate Golems Resource pack](https://www.curseforge.com/minecraft/texture-packs/accurate-golems/screenshots) by default.

**Comment by ACGaming** [comment]
Implemented in f82bba3d1f9ab6469f325cc7136c7a59016bebee.
